plugins/example_plugin.bash

```diff
@@ -79,6 +79,9 @@ restore_data() {
   timestamp_day_dir=${timestamp_dir%??????}
   if [ -e "/tmp/GPBACKUP_PLUGIN_LOG_TO_STDERR" ] ; then
     echo 'Some plugin warning' >&2
+  elif [ -e "/tmp/GPBACKUP_PLUGIN_FAIL" ] ; then
+  	pkill "gpbackup_helper"
```

Comment:
The tests seem to pass just fine without this line.

Also, this and the next line have spaces and tabs mixed together. Could you change them to spaces?

Reply:
> The tests seem to pass just fine without this line.
It is because `exit 1`, which made the gprestore fail anyway. It is redundant, and I've removed it.

> Also, this and the next line have spaces and tabs mixed together. Could you change them to spaces?
Updated